### PR TITLE
Add changelog entry for the addition of hasUDA to std.traits

### PR DESCRIPTION
**Pull request by Hackerpilot**
See https://github.com/D-Programming-Language/phobos/pull/3356

### CODE REVIEW

**Comment by Hackerpilot** [comment]
https://github.com/D-Programming-Language/phobos/pull/3356 was merged, so this should be merged as well.

File: changelog.dd [hunk 1]
```diff
@@ -39,6 +39,11 @@ $(LI $(LNAME2 array-bool-4733, The compiler now generates a warning when using a
 
 )
 
+$(BUGSTITLE Library Changes,
+$(LI $(LINK2 phobos/std_traits.html#.hasUDA, $(D hasUDA)) was added to help check
+	for the existance of user-defined attributes on symbols.)
+)
```
Comment (aG0aep6G) [inline changelog.dd]: existence

**Comment by CyberShadow** [comment]
@Hackerpilot ping

**Comment by Hackerpilot** [comment]
Fixed.

**Pull request merged**